config/bootstrap.rb

```diff
@@ -32,3 +32,10 @@ def perform
     add_transaction_tracer :perform, :category => :task
   end
 end
+
+require 'routemaster/mixins/log'
+include Routemaster::Mixins::Log
```

Comment:
Ah, sorry I thought you meant lazily check that inside `warn`, `info` calls. You meant checking that inside `_log` method. Yeah, that makes sense.